polyfill/lib/duration.mjs

```diff
@@ -305,11 +309,7 @@ export class Duration {
 
     let precalculatedPlainDateTime;
     const plainDateTimeOrRelativeToWillBeUsed =
```

Comment:
I opened a new issue for that primarily because this code will look entirely different after #2826 and in the spec text we probably want to just do whatever is the simplest since it will no longer be observable from user code.